deployer/src/snippeter.py

```diff
@@ -81,7 +81,7 @@ def get_email_for_config(config, analytics_statuses=None):
         for name, values in list(facets.items()):
             if name == "no_variables":
                 continue
-            keys = values.keys()
+            keys = list(values.keys())
```

Comment:
why?

Reply:
fix: dictionary iteration methods to make it compliant with python v3